core/templates/filters/string-utility-filters/normalize-whitespace.pipe.spec.ts

```diff
@@ -35,8 +35,6 @@ describe('Testing NormalizeWhitespacePipe', () => {
 
   it('should correctly normalize whitespace', () => {
     expect(nwp.transform('')).toEqual('');
-    expect(nwp.transform(null)).toEqual(null);
```

Comment:
why can it never be `null` or `undefined`?

Reply:
I confirmed this with @srijanreddy98 , these test cases along with one at line:37 needs to be removed here . Won't be  using this pipe with `' '`, `null` or `undefined` as arguments.